app/services/gtfs_graph.rb

```diff
@@ -84,26 +92,25 @@ def ssp_perform_async(trip_ids, agency_map, route_map, stop_map)
       ssps += ssp_trip
       # If chunk is big enough, create change payloads.
       if ssps.size >= CHANGE_PAYLOAD_MAX_ENTITIES
-        log  "  ssps: #{total} - #{total+ssps.size}"
+        log "  ssps: #{total} - #{total + ssps.size}"
         total += ssps.size
-        create_change_payloads(changeset, 'scheduleStopPair', ssps.map { |e| make_change_ssp(e) })
+        create_change_payloads(changeset, ssps)
         ssps = []
       end
     end
     # Create any trailing payloads
     if ssps.size > 0
-      log  "  ssps: #{total} - #{total+ssps.size}"
+      log "  ssps: #{total} - #{total + ssps.size}"
       total += ssps.size
-      create_change_payloads(changeset, 'scheduleStopPair', ssps.map { |e| make_change_ssp(e) })
+      create_change_payloads(changeset, ssps)
     end
-    log "Changeset apply"
-    t = Time.now
+    log 'Changeset apply'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.